lucille-core/src/main/java/com/kmwllc/lucille/stage/LookupCollection.java

```diff
@@ -0,0 +1,10 @@
+package com.kmwllc.lucille.stage;
+
+public interface LookupCollection<T> {
```

Comment:
why do we need this interface?
why wouldn't it suffice to use a HashSet for our purposes?